integrations/dotnet/install-driver.sh

```diff
@@ -38,11 +39,13 @@ else
         https://dot.net/v1/dotnet-install.sh
 
     bash ./integrations/dotnet/dotnet-install.sh -Channel 6.0 --install-dir .dotnet --no-path
+    bash ./integrations/dotnet/dotnet-install.sh -Channel 8.0 --install-dir .dotnet --no-path
 fi
 
 # /p required to get around https://github.com/dotnet/sdk/issues/12159
-./.dotnet/dotnet build mongo-csharp-driver /p:Platform="Any CPU" # platform needs a space when building
+./.dotnet/dotnet build mongo-csharp-driver /p:Platform="Any CPU" -c Debug # platform needs a space when building
 ./.dotnet/dotnet publish mongo-csharp-driver/tests/AstrolabeWorkloadExecutor \
+    -c Debug \
```

Comment:
In .net8 `dotnet publish` looks by default for `Release` config, we assumed that the default was `Debug`.

Reply:
Shouldn't we be building and testing release binaries?